### PR TITLE
Uplift to fix all vulnerabilities

### PR DESCRIPTION
**Pull request by jasonkshar**
Addressed all vulnerabilities as reported by `npm audit`. Many major changes were made as part of this:
* Uplifted Terra-Toolkit from 1.X to 5.X
  * v5 of Terra-Toolkit removed support for Nightwatch, so those tests are rewritten in WDIO
* Updated to babel 7.x
* Updated to webpack 4.x

### CODE REVIEW

**Comment by jasonkshar** [comment]
There is one outstanding peer dependency warning:
`npm WARN terra-toolkit@5.12.0 requires a peer of webpack@^4.30.0 but none is installed. You must install peer dependencies yourself.`

I don't think you can update webpack to 4.30.0 because of this issue: https://github.com/webpack/webpack/issues/8656

**Comment by jasonkshar** [comment]
Since the wdio tests require docker now, I'm wondering if there's something that needs to be configured for travis. Might have to reach out to the terra-toolkit maintainers for that.

**Comment by kolkheang** [comment]
Maybe we need to install Chrome using headless mode? 

https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-the-chrome-addon-in-the-headless-mode

**Comment by jasonkshar** [comment]
No bueno. That testing should all be contained within the docker container, I think.

**Comment by jasonkshar** [comment]
guess you needed to specify a few things:
* node 10
* docker-compose and Dockerfile

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ARG project_name=ascvd-risk-calculator
```
Comment (mjhenkes) [inline Dockerfile]: You can probably remove this line

**Comment by kolkheang** [comment]
@jasonkshar  could you squash all the commits when you merge?

@vibhutitripathi12 , @shriniketsarkar  could you review when you get a chance?

File: package.json [hunk 2]
```diff
@@ -11,67 +11,93 @@
...
+    "react-intl": "^2.9.0",
+    "terra-clinical-error-view": "^1.3.0"
   },
   "devDependencies": {
-    "babel-core": "^6.18.2",
-    "babel-jest": "^17.0.2",
-    "babel-loader": "^6.2.8",
+    "@babel/cli": "^7.6.4",
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
```
Comment (kolkheang) [inline package.json]: Are all of these @babel/plugin-proposal-* modules needed?

**Comment by jasonkshar** [comment]
I don't have permissions to merge, but isn't there an option to squash and merge when accepting this PR?

File: package.json (hunk 2 above)
Comment (jasonkshar) [reply package.json]: I think so. I used https://github.com/babel/babel-upgrade to go through it.

Comment (kolkheang) [reply package.json]: +1

**Comment by kolkheang** [comment]
@jasonkshar can you re-run `npm run build`?  It looks like a few files were deleted from the `/build/` dir.

**Comment by jasonkshar** [comment]
If you're referring to the `intl-polyfill-bundle.min.js` file, that file gets removed when I `npm run build` is executed.

**Comment by kolkheang** [comment]
@jasonkshar I think that is going to be a problem for the app since currently that file is being referenced in the `index.html`. 
https://github.com/cerner/ascvd-risk-calculator/blob/master/index.html#L11

**Comment by jasonkshar** [comment]
That was a problem then, couldn't trust `webpack-merge`

**Comment by shriniketsarkar** [comment]
@jasonkshar Sorry about the delay in reviewing it.  Thanks for working on this :) .

**Comment by kolkheang** [comment]
Thanks @jasonkshar for contributing!

**Pull request merged**